clang/lib/Interpreter/InterpreterValuePrinter.cpp

```diff
@@ -300,16 +640,17 @@ llvm::Expected<Expr *> Interpreter::ExtractValueFromExpr(Expr *E) {
 using namespace clang;
 
 // Temporary rvalue struct that need special care.
-REPL_EXTERNAL_VISIBILITY void *
+REPL_EXTERNAL_VISIBILITY extern "C" void *
```

Comment:
@AaronBallman I will need some help here. I think the windows issue we are seeing stems from here. If I move the `REPL_EXTERNAL_VISIBILITY` after `extern "C"` msvc errors out. If I keep it there I do not think it has an effect to export the symbol...

Reply:
Exactly...

Reply:
I resolved this by adding an `extern "C"` block...